### DISCUSSION TITLE
Automatic extension update

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#313] [Q&A (Community Support)] [answered]
Hi there are 2 problems with automatic update 
1. On Akeeba Backup update get error like this
`Extension updates for site #38 (********): failed installing update for package “Akeeba Backup for Joomla! package”. Guzzle error: Client error: `POST https://www.********/api/index.php/v1/panopticon/update` resulted in a `404 Not Found` response: {"errors":[{"title":"Resource not found","code":404}]}`
2. On Admin Tools update get error
`Extension updates for site #38 (********): failed installing update for package “Admin Tools package”. Joomla! reported an error: No updates`
but when login to page, I see there is update for the Admin Tools.

Don’t know the solution to a problem, any help, please?
Thank you

**Comment by nikosdion** [comment]
As per the error message, the API endpoint is not found which means that the plugin is disabled or not installed on your site.

**Comment by komir2004** [comment]
As I can see it is enabled
![Extensions-Manage](https://github.com/akeeba/panopticon/assets/17553018/c35c2cb7-b522-4d30-9547-bf36a990a8c0)

**Reply by Rolli1962** [reply]
And even access for public ?
![grafik](https://github.com/akeeba/panopticon/assets/60449692/edb83bfc-9c9f-40d4-abe0-99c18a6b3ca5)

**Comment by komir2004** [comment]
Yes 

![Plugins-Korisnici-2](https://github.com/akeeba/panopticon/assets/17553018/f4bee7e3-9efd-4989-a0ad-533012fc7ad7)

**Comment by nikosdion** [comment]
And the Akeeba Panopticon Connector component is installed and published?

**Comment by komir2004** [comment]
Yes it is https://github.com/akeeba/panopticon/discussions/313#discussioncomment-7599096

**Reply by Rolli1962** [reply]
Maybe this is something for the 'boss' ;-)
One hint from me as user - Nikosdion has written a very good description, can be found here in Wiki. Did you read and check the chapter of '[Connection](https://github.com/akeeba/panopticon/wiki/Managing-Sites)' ?

**Reply by nikosdion** [reply, marked as answer]
If your site redirects from http:// to https:// (or vice-versa), or from www to non-www (or vice-versa) make sure that the URL of the site you use is the one which will end being used after all redirections are done.

For example, our site (https://www.akeeba.com) redirects FROM http:// TO https://, and FROM non-www (akeeba.com) TO www (www.akeeba.com). Therefore, the API endpoint for our site is https://www.akeeba.com/api. If I were to use http://akeeba.com/api it would fail miserably because the redirections would end up screwing up the URL, therefore all API calls would result in a 404 route not found error. Sounds familiar, @komir2004 ?
